### DISCUSSION TITLE
클로저가 무엇인가여?

### DISCUSSION THREAD

**Discussion by gygy7151** [EasyAndBeauty/DailyReceipt-FE-WEB#21] [Q&A] [answered]
Jay가 알려준 링크 공유합니댱 ㅎ
https://intrepidgeeks.com/tutorial/core-javascript5-panel-cabinet

**Comment by jayjeong8** [comment, marked as answer]
링크 이거예요! 블루를 위한 선물 ~ https://ko.javascript.info/closure ~

**Reply by mdgarden** [reply]
헉 링크 공유 넘 감사합니다!!

**Reply by gygy7151** [reply]
고맙습니당 jay!!! 넘나리 쏘스윗해서 녹아여 진쨔
